docker/compose/docker-compose.yml

```diff
@@ -79,8 +79,11 @@ services:
     ports:
       - "12345:12345"
     command: segmentstore
+    privileged: true
     environment:
       WAIT_FOR: bookie1:3181,bookie2:3182,bookie3:3183,hdfs:8020
```

Comment:
Both things are possible. These will require changes to this file though.
1. Larger BK cluster: Add the extra bookies to the deployment and the WAIT_FOR list.
2. External BK cluster: Add the URL to the segmentstore parameters and do not start/wait for the bookies.